wiki/content/enterprise-features/index.md

```diff
@@ -703,3 +702,47 @@ Here's an example to run bulk loader with a key used to write encrypted data:
 ```bash
 dgraph bulk --encryption_key_file "./enc_key_file" -f data.json.gz -s data.schema --map_shards=1 --reduce_shards=1 --http localhost:8000 --zero=localhost:5080
 ```
+
+## Encrypted Backups
+
+Encrypted backups are a Enterprise feature that allow you to encrypt your backups and restore them. This documentation describes how to implement encryption into your binary backups
+
+### New flag “Encrypted” in manifest.json
```

Comment:
This section is an implementation detail. I dont think we need to document this.